### PR TITLE
SECURITY.md updates and ownership

### LINKED ISSUE #8171: Review recent security.md edits & add codeowners to ensure future changes are reviewed
## What is the Problem Being Solved?
Ensure security team reviews recent `security.md` edits so they know how to approach with the public.
Ensure future edits are reviewed by security team via `codeowners`

### PR DESCRIPTION
**Pull request by raphdev**
closes: #8171

## Description

This PR makes revisions to `SECURITY.md` to reflect currently supported versions. 

We also add a `CODEOWNERS` file and make `SECURITY.md` owned by the security team (@Agoric/sec). Since the team is ultimately accountable for the expectations set in this doc, we want to make sure modifications are reviewed by security for awareness and input.

### CODE REVIEW

File: SECURITY.md
Comment (raphdev) [inline SECURITY.md]: I'm not sure what the correct descriptor should be, should we include latest and pre-release tagged releases in https://github.com/Agoric/agoric-sdk/releases ?

File: CODEOWNERS
Comment (mhofman) [inline CODEOWNERS]: GH seem to be complaining about this

File: SECURITY.md
Comment (mhofman) [reply SECURITY.md]: Branch of that shape is definitely not correct, it should be `agoric-upgrade-*` **tags**, but as you mention without restrictions it would include outdated releases that may have known issues. Maybe something like 
> The current `master` branch and the `agoric-upgrade-*` tags corresponding to the latest [release and pre-release versions](https://github.com/Agoric/agoric-sdk/releases).

PS: We do have `release-*` branches, but those are mostly to stage the creation of these release tags, and may similarly become outdated like `release-pismo` now is.

**Review by mhofman — CHANGES_REQUESTED** [review]

File: CODEOWNERS
Comment (raphdev) [reply CODEOWNERS]: The permission changes are pending. This group needs write access to repo which will be done before merging.

**Review by ivanlei — APPROVED** [review]
LGTM

File: CODEOWNERS
Comment (mhofman) [inline CODEOWNERS]: Nit: new line missing

File: SECURITY.md
Comment (mhofman) [inline SECURITY.md:5]: Note: I don't expect us to ever have a latest pre-release tag that is older (aka ancestor commit) than the latest release tag, so this works, but it initially worried me about opening ourselves to reports on outdated pre-release tags. More precisely, given our release process, the latest pre-release tag should always be the same revision as the latest release tag, or newer when we're about to release an upgrade.

**Review by mhofman — APPROVED** [review]

**Pull request merged**
